mkosi

```diff
@@ -2345,6 +2350,7 @@ def install_arch(args: CommandLineArguments, root: str, do_run_build_script: boo
                             kernel_version=()
                             for f in /usr/lib/modules/*/vmlinuz
                             do
+                                [[ -e "$f" ]] || continue
```

Comment:
See commit message, when a glob doesn't have any matches, it returns the glob itself. That's what we're handling here.

Reply:
I'll add a comment and check against the glob itself which should be clearer.

Reply:
The better solution, since we are bash only, would be to set `shopt -s nullglob` (if I remember the shopt synxtax correctly) which makes empty globs truly empty.